functorch/csrc/BatchRulesBinaryOps.cpp

```diff
@@ -39,9 +37,11 @@ std::tuple<Tensor,optional<int64_t>> _binary_pointwise_batch_rule(
   auto tensor_is_logical_scalar = (tensor_logical_rank == 0 && tensor_batch_dim.has_value());
   auto other_is_logical_scalar = (other_logical_rank == 0 && other_batch_dim.has_value());
   if (tensor_is_logical_scalar && !other_is_logical_scalar) {
+    std::cout << "HERE 1" << std::endl;
     handleScalarTypePromotion(tensor_, other_);
   }
   if (other_is_logical_scalar && !tensor_is_logical_scalar) {
+    std::cout << "HERE 2" << std::endl;
```

Comment:
You probably want to delete this before I merge this?

Reply:
Sorry about that :) yes, this shouldn't be there